### ISSUE TITLE
Config file not working

### ISSUE THREAD

**Issue by mannidm** [lllyasviel/Fooocus#1801]
I read the troubleshooter and there is nothing about the config file issue I am having.

**Describe the problem**
I modified the config.txt file to what I wanted loaded for prompts and stuff and there is no error but the settings are not in place.
Here is my config file, looks like the tutorial and I see no issues. If I did anything wrong please let em know!
`
{
    "path_checkpoints": "S:\\fooocus\\Fooocus\\models\\checkpoints",
    "path_loras": "S:\\fooocus\\Fooocus\\models\\loras",
    "path_embeddings": "S:\\fooocus\\Fooocus\\models\\embeddings",
    "path_vae_approx": "S:\\fooocus\\Fooocus\\models\\vae_approx",
    "path_upscale_models": "S:\\fooocus\\Fooocus\\models\\upscale_models",
    "path_inpaint": "S:\\fooocus\\Fooocus\\models\\inpaint",
    "path_controlnet": "S:\\fooocus\\Fooocus\\models\\controlnet",
    "path_clip_vision": "S:\\fooocus\\Fooocus\\models\\clip_vision",
    "path_fooocus_expansion": "S:\\fooocus\\Fooocus\\models\\prompt_expansion\\fooocus_expansion",
    "path_outputs": "S:\\fooocus\\Fooocus\\outputs",
    "default_model": "realisticStockPhoto_v10.safetensors",
    "default_refiner": "",
    "default_refiner_switch": 0.6,
    "default_loras": [
        [
            "SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors",
            0.25
        ],
        [
            "None",
            0.8
        ],
        [
            "None",
            0.8
        ],
        [
            "None",
            0.8
        ],
        [
            "None",
            0.8
        ]
    ],
    "default_cfg_scale": 3.0,
    "default_sample_sharpness": 10.0,
    "default_sampler": "dpmpp_2m_sde_gpu",
    "default_scheduler": "karras",
    "default_styles": [
        "Fooocus V2",
        "Fooocus Photograph",
        "Fooocus Negative",
        "Fooocus Enhance",
        "Fooocus Sharp"
    ],
    "default_prompt_negative": "unrealistic, saturated, high contrast, big nose, painting, drawing, sketch, cartoon, anime, manga, render, CG, 3d, watermark, signature, label, Pubic hair",
    "default_prompt": "light brunette hair with loose curls, 21 years old, 5 feet tall, Blue eyes, smart, attractive, highly detailed, vibrant, dramatic, artistic,",
    "default_performance": "Quality",
    "default_advanced_checkbox": true,
    "default_max_image_number": 32,
    "default_image_number": 1,
    "checkpoint_downloads": {
        "realisticStockPhoto_v10.safetensors": "https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/realisticStockPhoto_v10.safetensors"
    },
    "lora_downloads": {
        "SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors": "https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors"
    },
    "embeddings_downloads": {},
    "available_aspect_ratios": [
        "704*1408",
        "704*1344",
        "768*1344",
        "768*1280",
        "832*1216",
        "832*1152",
        "896*1152",
        "896*1088",
        "960*1088",
        "960*1024",
        "1024*1024",
        "1024*960",
        "1088*960",
        "1088*896",
        "1152*896",
        "1152*832",
        "1216*832",
        "1280*768",
        "1344*768",
        "1344*704",
        "1408*704",
        "1472*704",
        "1536*640",
        "1600*640",
        "1664*576",
        "1728*576"
    ],
    "default_aspect_ratio": "1024*1024",
    "default_inpaint_engine_version": "v2.6",
    "default_cfg_tsnr": 7.0,
    "default_overwrite_step": -1,
    "default_overwrite_switch": -1,
    "example_inpaint_prompts": [
        "highly detailed face",
        "detailed girl face",
        "detailed man face",
        "detailed hand",
        "beautiful eyes"
    ]
}
`

**Full Console Log**
`
S:\fooocus>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py', '--preset', 'realistic']
Loaded preset: S:\fooocus\Fooocus\presets\realistic.json
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.1.861
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
Total VRAM 8192 MB, total RAM 16891 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce RTX 3060 Ti : native
VAE dtype: torch.bfloat16
Using pytorch cross attention
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection'}
Base model loaded: S:\fooocus\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors
Request to load LoRAs [['SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors', 0.25], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [S:\fooocus\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors].
Loaded LoRA [S:\fooocus\Fooocus\models\loras\SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors] for UNet [S:\fooocus\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors] with 788 keys at weight 0.25.
Loaded LoRA [S:\fooocus\Fooocus\models\loras\SDXL_FILM_PHOTOGRAPHY_STYLE_BetaV0.4.safetensors] for CLIP [S:\fooocus\Fooocus\models\checkpoints\realisticStockPhoto_v10.safetensors] with 264 keys at weight 0.25.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 1.21 seconds
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
`

**Comment by mashb1t** [comment]
As you're using `--preset realistic`, the preset overrides your config in specific settings. Please provide more context to what you're trying to achieve exactly and which config settings / element default values aren't set according to your expectations. Thanks!

**Comment by mannidm** [comment]
ok, well I wanted it to load my default settings, like picture format sharpness and my positive and negative starting prompts. If I cannot use the realistic preset, is there a way in the config to setup the same items as the preset? I dont know what all it changes. I just know that my presets are the things that I change evertime I start the app and was hoping to have them pre-set.

**Comment by mashb1t** [comment]
Please refer to the file `config_modification_tutorial` to see what can be changed and make sure you're using no comma after the last item

incorrect:
```
        "beautiful eyes"
    ],
}
```

correct:
```
        "beautiful eyes"
    ]
}
```

**Comment by mannidm** [comment]
For others looking for this info, the proper answer is to look at the presets folder realistic.json and verify the settings match on the items you do not want to change then just use run.bat to use your config. This appears to fix the issue, although the first time you run it if you dont have the default tensors it will download them.

**Comment by mashb1t** [comment]
Glad you found a solution 👍

**Comment by gsscoder** [comment]
If can be of any help to someone I was not able to let Fooocus read my new aspect ratios defined in `available_aspect_ratios`.
I updated directly `modules/config.py`:

```python
# ...
def add_ratio(x):
    a, b = x.replace('*', ' ').split(' ')[:2]
    a, b = int(a), int(b)
    g = math.gcd(a, b)
    return f'{a}×{b} <span style="color: grey;"> \U00002223 {a // g}:{b // g}</span>'

available_aspect_ratios.append('1080*1349')
available_aspect_ratios.append('1728*1728')

default_aspect_ratio = add_ratio(default_aspect_ratio)
available_aspect_ratios_labels = [add_ratio(x) for x in available_aspect_ratios]
# ...
```
